packages/calcite-components/src/components/list-item/list-item.scss

```diff
@@ -399,6 +399,13 @@
   }
 }
 
+.drag-container,
+.selection-container,
+.expanded-container,
+.actions-start {
```

Comment:
We can remove this from `actions-start`. It is already inside the bordered container.

Reply:
Weirdly, this doesn't seem to have worked anyway and I'm not seeing any related visual changes on action-start. I'm going to approve the PR, but we can still lose line 405.